src/main/java/shizuka/TodoList.java

```diff
@@ -0,0 +1,110 @@
+package shizuka;
+
+import java.util.ArrayList;
+
+public class TodoList {
+    static final String LINE_BREAK = "____________________\n";
+    static final String YOUR_TODO_LIST = "This is your todo list.\n";
+    static final String MARKED_TASK = "I have marked Task ";
+    static final String UNMARKED_TASK = "I have unmarked Task ";
+    static final String IN_LIST = " in your todo list.\n";
+    static final int MAX_DEADLINE_ARGS = -1;
+    static final int MAX_EVENT_ARGS = 3;
+    static final String BY_COMMAND = " /by ";
+    static final String FROM_COMMAND = " /from ";
+    static final String TO_COMMAND = " /to ";
+    private int numberOfTasks;
+    private final ArrayList<Todo> todos;
+
+    public TodoList(){
+        numberOfTasks = 0;
+        this.todos = new ArrayList<>();
+    }
+
+    private static String[] parseDeadline(String args) throws InvalidCommandFormatException {
+        String[] vars = args.split(BY_COMMAND, MAX_DEADLINE_ARGS);
+        if (vars.length != 2) {
+            throw new InvalidCommandFormatException();
+        }
+        return vars;
+    }
+
+    private static String[] parseEvent(String args) throws InvalidCommandFormatException {
+        String[] vars = new String[MAX_EVENT_ARGS];
+        String[] tempArray0 = args.split(FROM_COMMAND, 2);
+        vars[0] = tempArray0[0];
+        String[] tempArray1 = tempArray0[1].split(TO_COMMAND, 2);
+        vars[1] = tempArray1[0];
+        vars[2] = tempArray1[1];
```

Comment:
As mentioned before, perhaps the names could be more informative here, so the code can be understood directly?